### DISCUSSION TITLE
Salary by rank

### DISCUSSION THREAD

**Discussion by LucasHervieu** [Zrips/CMI#8816] [Q&A] [answered]
hello, is there an hourly wage per ranks?

**Comment by LucasHervieu** [comment, marked as answer]
I finally succeeded with “preward” with luckyperm groups but when you're an OP you see all the prewards, you just have to stop being an OP and be in a group and you're done.

So one point to improve: don't see all the prewards when you're OP but only when you're in a group.
now how do you make sure when you're an OP that you no longer receive all your wages...

**Reply by LucasHervieu** [reply]
@Zrips help please

**Comment by CloudeLecaw** [comment]
thats not possible. also your players never should get the op rank.

**Reply by LucasHervieu** [reply]
yes, it's normal. there's no way to avoid paying as an admin.
